src/core/main.cpp

```diff
@@ -528,6 +528,57 @@ int main( int argc, char * * argv )
 		}
 	}
 
+	// Test file argument before continuing
+	QFileInfo fileInfoLoad( fileToLoad );
+	QFileInfo fileInfoImport( fileToImport );
+	if( !fileToLoad.isEmpty() )
```

Comment:
`fileInfoLoad` should be instantiated in the `if( !fileToLoad.isEmpty() )` block; `fileInfoImport`, in the `if( !fileToImport.isEmpty() )` one.